src/YetORM/Repository.php

```diff
@@ -41,10 +41,10 @@ abstract class Repository
 
 
 	/** @param  NdbContext $database */
-	public function __construct(NdbContext $database)
+	public function __construct(NdbContext $database, ?Transaction $transaction = null)
```

Comment:
Please no PHP7 syntax yet.

```suggestion
	public function __construct(NdbContext $database, Transaction $transaction = null)
```